Directory.Build.props

```diff
@@ -2,6 +2,11 @@
     <Import Condition="Exists('obj/CommitCount.props')"
             Project="obj/CommitCount.props" />
     <PropertyGroup>
-        <CommitCount Condition="'$(CommitCount)' == ''">0</CommitCount>
+      <CommitCount Condition="'$(CommitCount)' == ''">0</CommitCount>
+      <RestoreSources>
```

Comment:
Should this be straight in the Directory.Build.Props? I think it should probably be in its own props file.

Reply:
Done...